### PR TITLE
[TYO-2017] Sponsor, program updates

### PR DESCRIPTION
**Pull request by apapadimoulis**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2017] Add Bluth Company as a sponsor`*

### CODE REVIEW

File: data/events/2017-tokyo.yml [hunk 1]
```diff
@@ -57,12 +57,14 @@ proposal_email: "proposals-tokyo-2017@devopsdays.org" # Put your proposal email
 # List all of your sponsors here along with what level of sponsorship they have.
 # Check data/sponsors/ to use sponsors already added by others.
 sponsors:
-  - id: opsgenie
```
Comment (mattstratton) [inline data/events/2017-tokyo.yml]: Keep the same sponsor name, but add a "url" field underneath it that points to the campaign-specific URL that Opsgenie gave you

File: data/sponsors/2017-tokyo-opsgenie.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+name: "OpsGenie"
+url: "http://www.opsgenie.com?utm_source=Events&utm_campaign=DevOpsDays_Tokyo_%20April&utm_medium=banner"
```
Comment (mattstratton) [inline data/sponsors/2017-tokyo-opsgenie.yml]: As mentioned, do not create this sponsor file, but just add the url. Check out https://github.com/devopsdays/devopsdays-web/blob/master/themes/devopsdays-theme/REFERENCE.md#sponsor-fields for more specifics.

**Comment by mattstratton** [comment]
In addition to the changes I listed above, just a heads up that this change would have resulted in this:

**Comment by apapadimoulis** [comment]
Whoops, thanks Matt.

Sorry it's not so clear... when you said "Please review the changes I submitted and address them before we merge this" --- do you mean that I should make changes, or that i need to somehow approve changes you made?

**Comment by mattstratton** [comment]
looks good to me now! Merging!

**Pull request merged**